org.kde.subtitlecomposer.json

```diff
@@ -66,7 +66,11 @@
 				{
 					"tag": "v5.76.0",
 					"type": "git",
-					"url": "https://invent.kde.org/frameworks/kross.git"
+					"url": "https://invent.kde.org/frameworks/kross.git",
```

Comment:
please use the tarballs, see for example lokalize, instead of git

Reply:
I decided not the change sources because this git source _may have been this way for a reason_.

I'll update this PR 👍

Reply:
I think at the time there was no tarball. Looks like frameworks only started getting added to download.k.o after 5.90, as listed here: https://download.kde.org/stable/frameworks/

Either that or I couldn't find a tarball.